DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -289,10 +296,10 @@ static function string _GetStartMap(int start_map_val, optional out string frien
             bShowInMenu=1;
             friendlyName = "NSF Defection (UNATCO HQ)";
             return "04_NYC_UNATCOHQ";
-        case 41:
+        case 42:
```

Comment:
_ChooseRandomStartMap had 41 in its list, which one was it getting before? unatco or streets? which one should it be giving?

Reply:
There's still a 41 case. In the current code it's there twice, making the 04_NYC_UNATCOHQ start unavailable and incorrectly giving 04_NYC_Street the start number 41 when it should come after.

Compare the current code:
```
        case 41:
            bShowInMenu=1;
            friendlyName = "NSF Defection (UNATCO HQ)";
            return "04_NYC_UNATCOHQ";
        case 41:
            friendlyName = "NSF Defection (Streets)";
            return "04_NYC_Street";
        case 42:
            friendlyName = "NSF Defection (Hotel)";
            return "04_NYC_Hotel";
```
to the new code:
```
        case 41:
            bShowInMenu=1;
            friendlyName = "NSF Defection (UNATCO HQ)";
            return "04_NYC_UNATCOHQ";
        case 42:
            friendlyName = "NSF Defection (Streets)";
            return "04_NYC_Street";
        case 43:
            friendlyName = "NSF Defection (Hotel)";
            return "04_NYC_Hotel";
```

Reply:
41 should have to be UNATCO, or else there's no way to start there (or Liberty Island) in M04.

Reply:
That is, 41 should have to be the start, not 42.